scripts/setup/upgrade-postgresql

```diff
@@ -58,6 +58,9 @@ pg_dropcluster "$UPGRADE_FROM" main
 apt remove -y "postgresql-$UPGRADE_FROM"
 if [ -n "$SCRIPTS_PATH" ]; then
     su postgres -c "$SCRIPTS_PATH/delete_old_cluster.sh"
+    if [ -f "$SCRIPTS_PATH/update_extensions.sql" ]; then
+        su postgres -c "psql $SCRIPTS_PATH/update_extensions.sql"
+    fi
```

Comment:
I think structurally it makes sense to run this before `delete_old_cluster.sh`?

Reply:
I don't really see how, but I'm happy to swap them around.